### PR TITLE
Redis session store

### PR DESCRIPTION
**Pull request by lokson**
Closes: https://github.com/kriasoft/nodejs-api-starter/issues/14

### CODE REVIEW

**Comment by koistya** [comment]
Cool, thanks! Since there is already a Redis client initialized in the app (`src/redis.js`) it might be a good idea to pass it into `connect-redis` store instead of `REDIS_URL`.

**Comment by lokson** [comment]
@koistya Welcome! I had same idea, but it was failing. Probably the reason was wrong syntax for currying call of redis-connect. It works now.

**Comment by lokson** [comment]
I don't know why travis is falling with message: `Error: Cannot find module 'arr-flatten'`. Should I add it to package.json?

**Comment by koistya** [comment]
@lokson no, that's fine, there was this error before (complaining about `arr-flatten`), dunno how to fix it yet.

File: src/app.js [hunk 1]
```diff
@@ -28,6 +30,7 @@ app.use(cors());
 app.use(bodyParser.urlencoded({ extended: true }));
 app.use(bodyParser.json());
 app.use(session({
+  store: new (connectRedis(session))({client: redis}),
```
Comment (koistya) [inline src/app.js]: I think ESLint may complain about missing spaces before/after `{`/`}` (`yarn lint`)

File: src/app.js [hunk 2]
```diff
@@ -13,6 +13,8 @@ import express from 'express';
 import cors from 'cors';
 import bodyParser from 'body-parser';
 import session from 'express-session';
+import redis from './redis';
```
Comment (koistya) [inline src/app.js]: ESLint would complain that a custom module import goes before global imports.

**Pull request merged**